RecoHGCal/TICL/plugins/TrackstersProducer.cc

```diff
@@ -102,9 +102,9 @@ TrackstersProducer::TrackstersProducer(const edm::ParameterSet& ps, const Tracks
     iterIndex_ = ticl::Trackster::TRKEM;
   else if (itername_ == "EM")
     iterIndex_ = ticl::Trackster::EM;
-  else if (itername_ == "TRK")
+  else if (itername_ == "Trk")
```

Comment:
Should we call it 'TrkHAD' to differentiate to 'TrkEM'?

Reply:
We decided to keep the same label used for the derived objects (`ticlTrackstersTrk`, `ticlMultiClustersFromTrackstersTrk`, ...) in order not modify the DQM index in this PR and trigger comparison differences.
We may change it later on.